packages/electron-app/src/main/index.ts

```diff
@@ -18,6 +19,10 @@ function createWindow () {
     process.env.ELECTRON_START_URL || 'http://127.0.0.1:3000'
   );
 
+  installExtension(REACT_DEVELOPER_TOOLS)
```

Comment:
My questions is: does this add the extension in the binary? In `yarn electron`? And if not, does it add something in the bundle size?

Reply:
ah good point - it would. we could separate the Dev and Prod environments with NODE_ENV, or we could do it on separate branches, as in, for things to get into master we need to prune this type of stuff out and we actually bundle from master but have a separate branch like 'development' which stays up to date with master but with all these extra logging dependencies?

Reply:
I think 2 branches is too much effort, a `if NODE_ENV else` would suffice.